ui/app/AppLayouts/Wallet/services/dapps/DappsConnectorSDK.qml

```diff
@@ -753,16 +754,17 @@ WalletConnectSDKBase {
                 console.warn(invalidDAppUrlError)
                 return
             }
-            const session = { url, name, iconUrl }
-            root.wcService.connectorDAppsProvider.addSession(JSON.stringify(session))
+
+            root.wcService.connectorDAppsProvider.addSession(url, name, iconUrl)
```

Comment:
@alexjba account param is missing here.

Reply:
It could be that's the reason, or maybe something else, but after I locally added the address I still was not able to connect to https://rarible.com vai browser connect (checked, the entry was in the db, but no connection was made on the dapp's side neither in the Stauts app).

Reply:
Will have a look

Reply:
> @alexjba account param is missing here.

It's missing, but there's nothing we can do now. The account is missing from the nim call. There's currently a workaround implemented below with a `TODO`. We're adding the session to the dapplist without waiting for the nim confirmation. Otherwise the dApp won't appear in the dApps list.



> It could be that's the reason, or maybe something else, but after I locally added the address I still was not able to connect to https://rarible.com vai browser connect (checked, the entry was in the db, but no connection was made on the dapp's side neither in the Stauts app).

I see that Rarible asks for a personal sign right after establishing the session. And the personal sign fails. But that expected I guess. We'll need to complete this PR https://github.com/status-im/status-desktop/pull/16047 first.